sdk/communication/Azure.Communication.PhoneNumbers/tests/SipRouting/SipRoutingClientLiveTests.cs

```diff
@@ -28,6 +28,20 @@ public SipRoutingClient InitializeTest()
             return client;
         }
 
+        [TearDown]
+        public void ClearConfiguration()
+        {
+            var client = CreateClient();
+            if (client.GetRoutesAsync().Result.Value.Count > 0)
+            {
+                client.SetRoutesAsync(new List<SipTrunkRoute>()).Wait();
+            }
+            if (client.GetTrunksAsync().Result.Value.Count > 0)
+            {
+                client.SetTrunksAsync(new List<SipTrunk>()).Wait();
```

Comment:
```suggestion
                client.SetRoutesAsync(new List<SipTrunkRoute>()).EnsureCompleted();
            }
            if (client.GetTrunksAsync().Result.Value.Count > 0)
            {
                client.SetTrunksAsync(new List<SipTrunk>()).EnsureCompleted();
```

Reply:
Used `EnsureCompleted()`.
To be able to run async tests the `Wait()` call was still needed.

var task = client.SetRoutesAsync(new List<SipTrunkRoute>());
task.Wait();
task.EnsureCompleted();

Reply:
Since this is just in the teardown which is not an async method - why not just use the sync versions (`SetTrunk(...)` and `GetTrunks(...)`) and avoid sync over async?

Reply:
As discussed offline, in the end used a solution with async teardown method.